tests/e2e/test_rosacli_cluster.go

```diff
@@ -332,6 +332,145 @@ var _ = Describe("Edit cluster",
 					To(ContainSubstring(
 						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
 			})
+
+		// Excluded until bug on OCP-73161 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := constants.DeleteProtectionEnabled
+				_, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=true",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the enable result from cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(clusterDetail.EnableDeleteProtection).To(Equal(deleteProtection))
+
+				By("Attempt to delete cluster with delete protection enabled")
+				defer func() {
+					_, err = clusterService.DeleteCluster(clusterID, "-y")
+					Expect(err).To(HaveOccurred())
+					textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+					Expect(textData).Should(ContainSubstring(
+						`Delete-protection has been activated on this cluster and 
+						it cannot be deleted until delete-protection is disabled`))
+				}()
+
+				By("Disable delete protection on the cluster")
+				deleteProtection = constants.DeleteProtectionDisabled
+				_, err = clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=false",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the disable result from cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(clusterDetail.EnableDeleteProtection).To(Equal(deleteProtection))
+			})
+
+		// Excluded until bug on OCP-74656 is resolved
+		It("can verify delete protection on a rosa cluster negative - [id:74656]",
```

Comment:
The tile MR contians the test case 71174 which I don't see automated anywhere, in place of that you missed this test case 74656